packages/python/plotly/js/widget.ts

```diff
@@ -1,7 +1,7 @@
 // Adapted from ../../../javascript/jupyterlab-plotly/src/Figure.ts for anywidget.
-import type { DOMWidgetModel } from "@jupyter-widgets/base";
-import _ from "lodash";
+import _ from "https://esm.sh/lodash-es@4.17.21";
 import Plotly from "plotly.js";
+/*global define */
```

Comment:
Shouldn't be a problem, but what is the purpose of define here?

Reply:
Just removed this - it was added to address an error I was seeing related to requirejs. I think this wasn't actually the required fix but I kept it in because I hadn't tested removing it yet. Just tested it and it looks like we're good!